pan/types.pan

```diff
@@ -148,37 +156,37 @@ function is_asndate = {
 };
 
 
-###################################################################
-# type_asndate
-#
-# AUTHOR: Charles Loomis <charles.loomis@cern.ch>
-###################################################################
+@documentation{
```

Comment:
simply remove this (it contains no actual added anything; except for the author field, but that is not the best way to keep this around.)